src/ape/managers/chain.py

```diff
@@ -530,10 +530,11 @@ def __getitem_int(self, index: int) -> ReceiptAPI:
 
     @__getitem__.register
     def __getitem_slice(self, indices: slice) -> List[ReceiptAPI]:
-        start, stop, step = (
+        # start, stop, step = (
+        start, stop = (
             indices.start or 0,
             indices.stop or len(self),
-            indices.step or 1,
+            # indices.step  or 1,
```

Comment:
Let's just not make this change. I don't know why your pre-commit is telling you to do this...